stylesheets/components/avatars.css

```diff
@@ -0,0 +1,67 @@
+.avatar-wrapper {
+  background-color: #e5e5e5d8;
```

Comment:
You could use a variable here for bg color, otherwise it looks good to merge!

Reply:
yes, I forgot about that. Thanks